experimental/hip/CMakeLists.txt

```diff
@@ -8,6 +8,19 @@
 set(IREE_PACKAGE_ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}")
 set(IREE_PACKAGE_ROOT_PREFIX "iree/experimental/hip")
 
+# TODO: maybe merge these variables with the equivalent in the ROCm driver.
+# The obstacle is that they are now experimental and should probably not polute
+# the main CMake file.
+set(IREE_HIP_DRIVER_TARGET_CHIP_DEFAULT "gfx908")
```

Comment:
also, why does the runtime driver need this? it really should not - maybe it's just for the cts? if so, we should move it there (and also fix that with multitargeting :) - @antiagainst

Reply:
There's a comment for this on https://github.com/openxla/iree/pull/16706/files#r1518437759 (should de-dup / land some of these related PRs)

Reply:
> and also fix that with multitargeting

So we build the tests for all chips and then the runtime should pick the correct binaries form the VM module?